Referenced by commit a2a5ae5b547, Fix Py's IDs user mapping: do not consider ShapeKeys' from here.
